server/api/model/plan_area_changes.js

```diff
@@ -0,0 +1,55 @@
+const Model = require('./base_model');
+const Log = require('../lib/log');
+const Exception = require('./exception');
+const { Bookshelf, Knex } = require('../service/database');
+const {	tags, tagDataRules } = require('../constants');
+
+class PlanAreaChanges extends Model {
```

Comment:
After the merge of the other PR, you sohuld merge it to this branch. I put some comments about this file on the other PR.

Reply:
Done.